netops_demo/golang/src/github.com/v3io/demos/functions/query/query.go

```diff
@@ -0,0 +1,93 @@
+package query
```

Comment:
Don't use this querier - it has bad practice like providing configuration in a string and does not follow the query spec as defined by Orit. Check out the latest querier @ https://github.com/v3io/tsdb-nuclio/blob/master/functions/query/query.go